pkg/oci/ccm.go

```diff
@@ -43,6 +44,13 @@ import (
 	"github.com/oracle/oci-cloud-controller-manager/pkg/oci/util"
 )
 
+const (
+	// RateLimitQPSDefault sets a value for the default queries per second.
+	RateLimitQPSDefault = 1.0
```

Comment:
@jhorwit2 Do you have any insight into appropriate values for these constants?

Reply:
@harveylowndes After talking to @jhorwit2 offline it seems like a value of `20.0` should be ok here.